### PR TITLE
PWX-34145: Exposing Portworx struct

### PR DESCRIPTION
**Pull request by yjoshi-px**
**What this PR does / why we need it**:
Exposing portworx struct and getKvdbMap function

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by piyush-nimbalkar — CHANGES_REQUESTED** [review]
Just pasting the comment from our slack conversation for everyone's ref - 

Let's not treat the `portworx` struct as an util if we are going to take some of the functions from it and make them public for that purpose. That is not how we should be changing the code.
Let’s do one of the following -
- Let’s create static utility methods like suggested above.
- Let’s create a new struct which exposes these public methods. We embed that struct in the portworx struct so the operator code does not change. Your test code can use that struct directly.

File: drivers/storage/portworx/util/util.go [hunk 1]
```diff
@@ -247,6 +249,10 @@ const (
 	// VsphereInstallModeLocal env value for Vsphere 'local' install
 	VsphereInstallModeLocal = "local"
 )
+const (
+	// pxEntriesKey is key which holds all the bootstrap entries
+	pxEntriesKey = "px-entries"
```
Comment (piyush-nimbalkar) [inline drivers/storage/portworx/util/util.go:254]: Can you completely remove the duplicate `pxEntriesKey` constant from status.go and replace all occurrences with this one?

File: drivers/storage/portworx/util/util.go [hunk 2]
```diff
@@ -1570,3 +1576,49 @@ func GetTLSCipherSuites(cluster *corev1.StorageCluster) (string, error) {
...
+	kvdbNodeMap := make(map[string]*kvdb_api.BootstrapEntry)
+	if cluster.Spec.Kvdb != nil && cluster.Spec.Kvdb.Internal {
+		clusterID := GetClusterID(cluster)
+		strippedClusterName := strings.ToLower(ConfigMapNameRegex.ReplaceAllString(clusterID, ""))
+		cmName := fmt.Sprintf("%s%s", InternalEtcdConfigMapPrefix, strippedClusterName)
+
+		cm := &v1.ConfigMap{}
+		err := k8sClient.Get(context.TODO(), types.NamespacedName{
+			Name:      cmName,
+			Namespace: "kube-system",
```
Comment (piyush-nimbalkar) [inline drivers/storage/portworx/util/util.go:1592]: Let's redeclare the constant `BootstrapCloudDriveNamespace` in this file and change the existing references to it.

**Review by piyush-nimbalkar — APPROVED** [review]
LGTM after the requested changes.

File: drivers/storage/portworx/util/util.go (hunk 2 above)
Comment (yjoshi-px) [reply drivers/storage/portworx/util/util.go]: Updated

File: drivers/storage/portworx/util/util.go (hunk 1 above)
Comment (yjoshi-px) [reply drivers/storage/portworx/util/util.go]: Updated

**Pull request merged**